### PR TITLE
Readme updated with step by step expected visual result on the scanner.

### PR DESCRIPTION
**Pull request by pan-**
Readme for the following examples have been updated:
* BLE_BatteryLevel
* BLE_Beacon
* BLE_Button
* BLE_EddystoneService
* BLE_GAPButton
* BLE_HeartRate
* BLE_LED
* BLE_Thermometer
* BLE_URIBeacon

### CODE REVIEW

**Comment by pan-** [comment]
@adbridge Could you review this please ?

File: BLE_BatteryLevel/readme.md [hunk 1]
```diff
@@ -23,13 +23,46 @@ Building instructions for all samples are in the [main readme](https://github.co
...
+    **figure 5** How to register to notifications using Master Control Panel 4.0.5
+
+
 1. You should see the battery level value change every half second. It begins at 50, goes up to 100 (in steps of 1), resets to 20 and so on.
 
+    ![](img/notifications.png)
+
+    **figure 6** Notifications view using Master Control Panel 4.0.5
+
 If you can see the characteristic, and if its value incrementing correctly, the application is working properly.
```
Comment (adbridge) [inline BLE_BatteryLevel/readme.md]: Should say .. and if its value is incrementing correctly

File: BLE_BatteryLevel/readme.md [hunk 2]
```diff
@@ -23,13 +23,46 @@ Building instructions for all samples are in the [main readme](https://github.co
 
 ## Checking for success
 
+**Note:** Screens captures depicted below shows what is expected from this example if the scanner used is *nRF Master Control Panel* version 4.0.5. If you encounter any difficulties to use another scanner or another version of *nRF Master Control Panel*, you might need to refer to the manual of your scanner.
```
Comment (adbridge) [inline BLE_BatteryLevel/readme.md]: If you encounter any difficulties consider trying another scanner or another version of nRF Master Control Panel. Alternative scanners may require reference to their manuals.

Comment (adbridge) [inline BLE_BatteryLevel/readme.md]: The above comment is common across the examples so you will probably need to update it in each.

File: BLE_LED/readme.md [hunk 3]
```diff
@@ -31,14 +31,63 @@ Building instructions for all samples are in the [main readme](https://github.co
...
+    **figure 4** Representation of the Led service using Master Control Panel 4.0.5
+
+1. Open the write pannel of the *LED state* characteristic.
+
+    ![](img/write_characteristic.png)
+
+    **figure 5** How to read and write a characteristic value using Master Control Panel 4.0.5
+
+
 1. The characteristic accept a 1-bytes value:
```
Comment (adbridge) [inline BLE_LED/readme.md]: The characteristic accepts a 1 byte value

**Comment by pan-** [comment]
Thank you for your comments Anna, I have updated the PR accordingly.

File: BLE_URIBeacon/readme.md [hunk 4]
```diff
@@ -48,8 +46,44 @@ binaries and hex files.
...
+
+1. Find your device; it should advertise the URL you have set.
+
+    ![](img/result.png)
+
+    **figure 5** Display of URL advertised by your beacon using the *Physical Web* application version 0.1.856 on Android.
+
 
 **Please note that the URIBeacon spec requires the URIBeacon app to remain in
 config mode for the first 60 seconds before switching to being a beacon. So if
```
Comment (adbridge) [inline BLE_URIBeacon/readme.md]: LGTM

**Comment by pan-** [comment]
I merge.

**Pull request merged**